schemas/com.snowplowanalytics.snowplow.badrows/enrichment_failures/jsonschema/1-0-0

```diff
@@ -0,0 +1,311 @@
+                      "value": {
+                        "type": [ "string", "null" ],
+                        "description": "Stringified representation of the value which did not meet the enrichment's expectations"
+                      },
+                      "expectation": {
+                        "type": "string",
+                        "description": "Description of the expectation which was not met"
+                      }
+                    },
+                    "required": ["field", "value", "expectation" ],
+                    "additionalProperties": false
+                  },
+                  {
+                    "type": "object",
+                    "description": "Error which was internal to the enrichment regarding the enrichment's output data, this is a schema validation error",
+                    "properties": {
+                      "errors": {
+                        "type": "array",
+                        "minItems": 1,
+                        "items": {
+                          "type": "object",
+                          "properties": {
+                            "path": {
+                              "type": "string",
+                              "description": "JSON Path to an issue in the faulty JSON Schema"
+                            },
+                            "message": {
+                              "type": "string",
+                              "description": "Human-readable message describing the issue with the schema"
+                            }
+                          },
+                          "required": [ "path", "message" ],
+                          "additionalProperties": false
+                        }
+                      }
+                    },
+                    "required": [ "errors" ],
+                    "additionalProperties": false
+                  },
+                  {
+                    "type": "object",
+                    "description": "Error which was external to the enrichment, e.g. a connection issue",
+                    "properties": {
+                      "error": {
+                        "type": "string",
+                        "description": "Error that occurred"
+                      }
+                    },
+                    "required": [ "error" ],
+                    "additionalProperties": false
+                  }
+                ]
+              }
+            },
+            "required": [ "enrichment", "message" ],
+            "additionalProperties": false
+          }
+        }
+      }
+    },
+    "payload": {
+      "type": "object",
+      "description": "The payload that resulted in enrichment failures",
+      "properties": {
+        "trackerProtocolPayload": {
+          "type": "object",
+          "description": "The tracker protocol payload that resulted in enrichment failures",
+          "properties": {
+            "app_id": { "type": ["string", "null"] },
```

Comment:
These don't look like tracker protocol properties. More like `enrichedEvent` (though it is not technically enriched yet) and I'd expect 134 properties here, but see only about 80.

Reply:
yes they're taken from the tracker protocol payload transformation that's why there are 80 and not more

Reply:
Can we maybe find more descriptive name then?

I actually think raw tracker protocol probably would be a better fit because it represents raw/original/untouched event. But maybe it will also be harder to query for analysts.